### DISCUSSION TITLE
Yamaha RX-V773 net radio always buffering

### DISCUSSION THREAD

**Discussion by BitasLT** [coffeegreg/YTuner#34] [Q&A] [answered]
Hello
need help or advice

my devices: 
1- receiver Yamaha RX-V773 
2- X96mini with Armbian 20.10 Arm-64 bullseye current 5.9.0 desktop like a server

actions: 
* I run the ytuner 1.2.3 program via x96 mini and go to net radio via yamaha and it shows three options: my station/browse radio/bookmarks, I connect to any internet radio and it always shows buffering and after 1 or 2 minutes it returns to the original radio window . 
* after starting the windows version of ytuner, the situation is the same
* In the Yamaha receiver, I change the IP as shown by the program

my problem: 
* after turning on any radio, it always shows buffering

**Comment by TheBossME** [comment]
1st. Have you changed DNS settings in your AVR ?

**Reply by BitasLT** [reply]
I think so
ytuner show: 
* DNS service: listening on: 192.168.0.139:53
* Web service: listening on: 192.168.0.139:80

* receiver ip adress- DNS server(P): 192.168.0.139

![IMG_20241020_133904](https://github.com/user-attachments/assets/e9d7ec9c-2fe5-4fac-a62b-599a78c538ae)
![IMG_20241020_134013](https://github.com/user-attachments/assets/d98efd53-6238-4ae7-ada1-2373ad779401)
![IMG_20241020_134054](https://github.com/user-attachments/assets/469a88e8-926e-4838-bcc1-9d4a78ee3fa7)
![IMG_20241020_134929](https://github.com/user-attachments/assets/97233fe8-995a-446b-b2e0-0c91178d1346)

**Reply by TheBossME** [reply]
Looked properly set. Have you unplugged the AVR from network and disconnect from Power voltage ?

My AVR buffered old settings. 

https://de.yamaha.com/de/support/updates/rx-a720-820_en.html

Which Firmware have you installed ?

**Reply by BitasLT** [reply]
update is the latest 1.98. I will try to turn it off
from the power and from the Internet. secondly i will try factory reset

**Reply by BitasLT** [reply]
disconnecting the electric, internet cable and factory settings did not help

**Reply by BitasLT** [reply, marked as answer]
WORKING!!!  I think it's my unknown or a mistake.
 solution: I changed Default Gateway from 192.168.182.1 to 192.168.0.1 and now it works

**Comment by coffeegreg** [comment]
@BitasLT 
Of course. :+1:  Your AVR's IP address (192.168.0.170) and your subnet mask (255.255.255.0 = /24) point to one class "C" subnet and your gateway must also be in this subnet (192.168.0.x).  :+1: 
I'm glad that YTuner also works properly for you. :smiley: :speaker: :notes: 

@TheBossME 
Beatrice, thank you for always being there to help. :tulip: I just got home. It was a beautiful and warm day. Good for long walks... :walking_man:
